jobsdb/jobsdb.go

```diff
@@ -2778,34 +2778,34 @@ func (jd *HandleT) backupDSLoop(ctx context.Context) {
 }
 
 //backupDS writes both jobs and job_staus table to JOBS_BACKUP_STORAGE_PROVIDER
-func (jd *HandleT) backupDS(backupDSRange dataSetRangeT) bool {
+func (jd *HandleT) backupDS(ctx context.Context, backupDSRange *dataSetRangeT) error {
 	// return after backing up aboprted jobs if the flag is turned on
 	// backupDS is only called when BackupSettings.BackupEnabled is true
 	if jd.BackupSettings.FailedOnly {
 		jd.logger.Info("[JobsDB] ::  backupDS: starting backing up aborted")
-		_, err := jd.backupTable(backupDSRange, false)
+		_, err := jd.backupTable(ctx, backupDSRange, false)
 		if err != nil {
 			jd.logger.Errorf("[JobsDB] :: Failed to backup aborted jobs table %v. Err: %v", backupDSRange.ds.JobStatusTable, err)
-			return false
+			return err
```

Comment:
Good idea, but actually caller function is printing the log with table name details itself. So, to avoid redundant contents in error log, I am returning just `err`.

Reply:
removed `jd.logger.Errorf` from `backupDS` func. Since, caller function is printing the logs anyways.